### ISSUE TITLE
The download links are broken?

### ISSUE THREAD

**Issue by TheVanilla** [Gictorbit/photoshopCClinux#119]
During the installation it will go through the steps and install vcredist. Once that is done it will say it was saved and move on to downloading photoshopCC-V19.1.6-2018x64.tgz at which point the installer will give this error: 
`ownloading photoshopCC-V19.1.6-2018x64.tgz ...
using curl to download photoshopCC-V19.1.6-2018x64.tgz
% Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
Dload  Upload   Total   Spent    Left  Speed
0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0curl: (6) Could not resolve host: victor.poshtiban.io
downloading photoshopCC-V19.1.6-2018x64.tgz ...
using curl to download photoshopCC-V19.1.6-2018x64.tgz
% Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
Dload  Upload   Total   Spent    Left  Speed
0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0curl: (6) Could not resolve host: victor.poshtiban.io
downloading photoshopCC-V19.1.6-2018x64.tgz ...
using curl to download photoshopCC-V19.1.6-2018x64.tgz
% Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
Dload  Upload   Total   Spent    Left  Speed
0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0curl: (6) Could not resolve host: victor.poshtiban.io
error: sorry something went wrong during download photoshopCC-V19.1.6-2018x64.tgz`
I am a beginner in using linux so I am not sure if I am doing something wrong or not but the download link being broken is the best explanation I've come up with.

**Comment by itsbek** [comment]
you're not doing anything wrong, it's a curl stopping the download for not being able to verify if it's secure which resulted in an error. I thought I could edit the script where it calls curl and pass the `-k` flag to skip the security check but not that pro to understand bash scripts...
anyway if you found a solution, would love to hear it out or @Gictorbit is the only one who can guide us through 

edit: I was scrolling through the issues and found the solution answered in [here](https://github.com/Gictorbit/photoshopCClinux/issues/163#issuecomment-1041583999)

**Comment by opiodaniel** [comment]
downloading photoshopCC-V19.1.6-2018x64.tgz ...
using curl to download photoshopCC-V19.1.6-2018x64.tgz
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0curl: (6) Could not resolve host: victor.poshtiban.io
downloading photoshopCC-V19.1.6-2018x64.tgz ...
using curl to download photoshopCC-V19.1.6-2018x64.tgz
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0curl: (6) Could not resolve host: victor.poshtiban.io
downloading photoshopCC-V19.1.6-2018x64.tgz ...
using curl to download photoshopCC-V19.1.6-2018x64.tgzs
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0curl: (6) Could not resolve host: victor.poshtiban.io
error: sorry something went wrong during download photoshopCC-V19.1.6-2018x64.tgz

i need help here. even after making the change inside the script file in line 197 i still get the above error.

**Comment by JoaoLucasMoraisOrtiz** [comment]
> downloading photoshopCC-V19.1.6-2018x64.tgz ... using curl to download photoshopCC-V19.1.6-2018x64.tgz % Total % Received % Xferd Average Speed Time Time Time Current Dload Upload Total Spent Left Speed 0 0 0 0 0 0 0 0 --:--:-- --:--:-- --:--:-- 0curl: (6) Could not resolve host: victor.poshtiban.io downloading photoshopCC-V19.1.6-2018x64.tgz ... using curl to download photoshopCC-V19.1.6-2018x64.tgz % Total % Received % Xferd Average Speed Time Time Time Current Dload Upload Total Spent Left Speed 0 0 0 0 0 0 0 0 --:--:-- --:--:-- --:--:-- 0curl: (6) Could not resolve host: victor.poshtiban.io downloading photoshopCC-V19.1.6-2018x64.tgz ... using curl to download photoshopCC-V19.1.6-2018x64.tgzs % Total % Received % Xferd Average Speed Time Time Time Current Dload Upload Total Spent Left Speed 0 0 0 0 0 0 0 0 --:--:-- --:--:-- --:--:-- 0curl: (6) Could not resolve host: victor.poshtiban.io error: sorry something went wrong during download photoshopCC-V19.1.6-2018x64.tgz
> 
> i need help here. even after making the change inside the script file in line 197 i still get the above error.

The link is broken. You can confirm this by visiting the website mentioned in the script: https://victor.poshtiban.io/p/gictor/photoshopCC/photoshopCC-V19.1.6-2018x64.tgz

Edit 1:
I found a new link to the replacement.tgz, which is: https://drive.google.com/uc?id=1AD_a6XfaiFY_eivEMdCajvjxRC-1P-NY/

Edit 2: the link to the photoshop: https://drive.google.com/uc?id=1kDVL7xjcyxlQD7DTo5QI71KkXg1sfT1t

**Comment by opiodaniel** [comment]
Thanx for the update.👌. Anyway is there any link that you recommend me to
follow to help me download Photoshop.

**Comment by g1i7ch** [comment]
anyone have an updated link for the replacement.tgz?

**Comment by KoustubhJain** [comment]
> > downloading photoshopCC-V19.1.6-2018x64.tgz ... using curl to download photoshopCC-V19.1.6-2018x64.tgz % Total % Received % Xferd Average Speed Time Time Time Current Dload Upload Total Spent Left Speed 0 0 0 0 0 0 0 0 --:--:-- --:--:-- --:--:-- 0curl: (6) Could not resolve host: victor.poshtiban.io downloading photoshopCC-V19.1.6-2018x64.tgz ... using curl to download photoshopCC-V19.1.6-2018x64.tgz % Total % Received % Xferd Average Speed Time Time Time Current Dload Upload Total Spent Left Speed 0 0 0 0 0 0 0 0 --:--:-- --:--:-- --:--:-- 0curl: (6) Could not resolve host: victor.poshtiban.io downloading photoshopCC-V19.1.6-2018x64.tgz ... using curl to download photoshopCC-V19.1.6-2018x64.tgzs % Total % Received % Xferd Average Speed Time Time Time Current Dload Upload Total Spent Left Speed 0 0 0 0 0 0 0 0 --:--:-- --:--:-- --:--:-- 0curl: (6) Could not resolve host: victor.poshtiban.io error: sorry something went wrong during download photoshopCC-V19.1.6-2018x64.tgz
> > i need help here. even after making the change inside the script file in line 197 i still get the above error.
> 
> The link is broken. You can confirm this by visiting the website mentioned in the script: https://victor.poshtiban.io/p/gictor/photoshopCC/photoshopCC-V19.1.6-2018x64.tgz
> 
> Edit 1: I found a new link to the replacement.tgz, which is: https://drive.google.com/uc?id=1AD_a6XfaiFY_eivEMdCajvjxRC-1P-NY/
> 
> Edit 2: the link to the photoshop: https://drive.google.com/uc?id=1kDVL7xjcyxlQD7DTo5QI71KkXg1sfT1t

Sorry to bug you, but the replacement link isn't working anymore

**Comment by joshuachris2001** [comment]
[JoaoLucasMoraisOrtiz](https://github.com/JoaoLucasMoraisOrtiz)'s link works perfectly
> Edit 2: the link to the photoshop: https://drive.google.com/uc?id=1kDVL7xjcyxlQD7DTo5QI71KkXg1sfT1t


Thanks! 👍

**Comment by Grphq1** [comment]
> > downloading photoshopCC-V19.1.6-2018x64.tgz ... using curl to download photoshopCC-V19.1.6-2018x64.tgz % Total % Received % Xferd Average Speed Time Time Time Current Dload Upload Total Spent Left Speed 0 0 0 0 0 0 0 0 --:--:-- --:--:-- --:--:-- 0curl: (6) Could not resolve host: victor.poshtiban.io downloading photoshopCC-V19.1.6-2018x64.tgz ... using curl to download photoshopCC-V19.1.6-2018x64.tgz % Total % Received % Xferd Average Speed Time Time Time Current Dload Upload Total Spent Left Speed 0 0 0 0 0 0 0 0 --:--:-- --:--:-- --:--:-- 0curl: (6) Could not resolve host: victor.poshtiban.io downloading photoshopCC-V19.1.6-2018x64.tgz ... using curl to download photoshopCC-V19.1.6-2018x64.tgzs % Total % Received % Xferd Average Speed Time Time Time Current Dload Upload Total Spent Left Speed 0 0 0 0 0 0 0 0 --:--:-- --:--:-- --:--:-- 0curl: (6) Could not resolve host: victor.poshtiban.io error: sorry something went wrong during download photoshopCC-V19.1.6-2018x64.tgz
> > i need help here. even after making the change inside the script file in line 197 i still get the above error.
> 
> The link is broken. You can confirm this by visiting the website mentioned in the script: https://victor.poshtiban.io/p/gictor/photoshopCC/photoshopCC-V19.1.6-2018x64.tgz
> 
> Edit 1: I found a new link to the replacement.tgz, which is: https://drive.google.com/uc?id=1AD_a6XfaiFY_eivEMdCajvjxRC-1P-NY/
> 
> Edit 2: the link to the photoshop: https://drive.google.com/uc?id=1kDVL7xjcyxlQD7DTo5QI71KkXg1sfT1t

replacement link isn't working for me

**Comment by Raytreq** [comment]
> > > downloading photoshopCC-V19.1.6-2018x64.tgz ... using curl to download photoshopCC-V19.1.6-2018x64.tgz % Total % Received % Xferd Average Speed Time Time Time Current Dload Upload Total Spent Left Speed 0 0 0 0 0 0 0 0 --:--:-- --:--:-- --:--:-- 0curl: (6) Could not resolve host: victor.poshtiban.io downloading photoshopCC-V19.1.6-2018x64.tgz ... using curl to download photoshopCC-V19.1.6-2018x64.tgz % Total % Received % Xferd Average Speed Time Time Time Current Dload Upload Total Spent Left Speed 0 0 0 0 0 0 0 0 --:--:-- --:--:-- --:--:-- 0curl: (6) Could not resolve host: victor.poshtiban.io downloading photoshopCC-V19.1.6-2018x64.tgz ... using curl to download photoshopCC-V19.1.6-2018x64.tgzs % Total % Received % Xferd Average Speed Time Time Time Current Dload Upload Total Spent Left Speed 0 0 0 0 0 0 0 0 --:--:-- --:--:-- --:--:-- 0curl: (6) Could not resolve host: victor.poshtiban.io error: sorry something went wrong during download photoshopCC-V19.1.6-2018x64.tgz
> > > i need help here. even after making the change inside the script file in line 197 i still get the above error.
> > 
> > 
> > The link is broken. You can confirm this by visiting the website mentioned in the script: https://victor.poshtiban.io/p/gictor/photoshopCC/photoshopCC-V19.1.6-2018x64.tgz
> > Edit 1: I found a new link to the replacement.tgz, which is: https://drive.google.com/uc?id=1AD_a6XfaiFY_eivEMdCajvjxRC-1P-NY/
> > Edit 2: the link to the photoshop: https://drive.google.com/uc?id=1kDVL7xjcyxlQD7DTo5QI71KkXg1sfT1t
> 
> replacement link isn't working for me

Same here

**Comment by Raytreq** [comment]
> > > > downloading photoshopCC-V19.1.6-2018x64.tgz ... using curl to download photoshopCC-V19.1.6-2018x64.tgz % Total % Received % Xferd Average Speed Time Time Time Current Dload Upload Total Spent Left Speed 0 0 0 0 0 0 0 0 --:--:-- --:--:-- --:--:-- 0curl: (6) Could not resolve host: victor.poshtiban.io downloading photoshopCC-V19.1.6-2018x64.tgz ... using curl to download photoshopCC-V19.1.6-2018x64.tgz % Total % Received % Xferd Average Speed Time Time Time Current Dload Upload Total Spent Left Speed 0 0 0 0 0 0 0 0 --:--:-- --:--:-- --:--:-- 0curl: (6) Could not resolve host: victor.poshtiban.io downloading photoshopCC-V19.1.6-2018x64.tgz ... using curl to download photoshopCC-V19.1.6-2018x64.tgzs % Total % Received % Xferd Average Speed Time Time Time Current Dload Upload Total Spent Left Speed 0 0 0 0 0 0 0 0 --:--:-- --:--:-- --:--:-- 0curl: (6) Could not resolve host: victor.poshtiban.io error: sorry something went wrong during download photoshopCC-V19.1.6-2018x64.tgz
> > > > i need help here. even after making the change inside the script file in line 197 i still get the above error.
> > > 
> > > 
> > > The link is broken. You can confirm this by visiting the website mentioned in the script: https://victor.poshtiban.io/p/gictor/photoshopCC/photoshopCC-V19.1.6-2018x64.tgz
> > > Edit 1: I found a new link to the replacement.tgz, which is: https://drive.google.com/uc?id=1AD_a6XfaiFY_eivEMdCajvjxRC-1P-NY/
> > > Edit 2: the link to the photoshop: https://drive.google.com/uc?id=1kDVL7xjcyxlQD7DTo5QI71KkXg1sfT1t
> > 
> > 
> > replacement link isn't working for me
> 
> Same here

Oh nevermind, I've just found a working link from an another issue (https://drive.google.com/file/d/1AD_a6XfaiFY_eivEMdCajvjxRC-1P-NY/view?usp=sharing)
